pkg/handler/crud.go

```diff
@@ -272,6 +279,29 @@ func (c *crud) SetFlagEnabledState(params flag.SetFlagEnabledParams) middleware.
 	return resp
 }
 
+func (c *crud) RestoreFlag(params flag.RestoreFlagParams) middleware.Responder {
+	f := &entity.Flag{}
+	if err := getDB().Unscoped().First(f, params.FlagID).Error; err != nil {
```

Comment:
since it's primarily used at the front page with the flags already loaded with tags, I think it's better to preload tags.
```suggestion
	if err := entity.PreloadFlagTags(getDB().Unscoped()).First(f, params.FlagID).Error; err != nil {
```